### ISSUE TITLE
TERRA-80 ⁃ data "astra_access_list" returns error when AstraDB does not have access list entries

### ISSUE THREAD

**Issue by giannoul** [datastax/terraform-provider-astra#171]
### Terraform Version
Terraform v1.1.8

### Affected Resource(s)
Please list the resources as a list, for example:

* data.astra_access_list

### Terraform Configuration Files
```hcl
data "astra_access_list" "dev" {
  database_id = "xxxxxx"
}
```

### Expected Behavior
It should return the list of the allowed rules, in this case null, empty array etc

### Actual Behavior
```
│ Error: json: cannot unmarshal string into Go value of type astra.Errors
│ 
│   with data.astra_access_list.dev,
│   on main.tf line 1, in data "astra_access_list" "dev":
│    1: data "astra_access_list" "dev" {
│ 
```

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:

1. `terraform plan` by using a `database_id ` of an AstraDb that does not yet have an access list:
![image](https://user-images.githubusercontent.com/9934202/192856097-2faea079-fa36-40c5-a485-08b686e13211.png)

### Important Factoids
The AstraDB does not yet have an access list.

The above error blocks us from utilizing the `data` resource `astra_access_list` when  managing multiple databases where some of them do have access list while others don't.

┆Issue is synchronized with this [Jira Task](https://datastax-oss.atlassian.net/browse/TERRA-80) by [Unito](https://www.unito.io)
┆friendlyId: TERRA-80
┆priority: Major

**Comment by emerkle826** [comment]
@giannoul This should be fixed with [d0303d3](https://github.com/datastax/terraform-provider-astra/commit/d0303d3e40475df1dfe2d1b7253c4cbc15dca697), which is being released as v2.1.6 shortly.

**Comment by giannoul** [comment]
Just used the v2.1.6 and the above has been fixed. Thank you @emerkle826
